flang/lib/Semantics/check-omp-structure.cpp

```diff
@@ -3127,43 +3219,14 @@ void OmpStructureChecker::CheckReductionModifier(
   }
 }
 
-void OmpStructureChecker::CheckIntentInPointerAndDefinable(
```

Comment:
The checks from this function already existed in separate functions.  They are used instead.